src/alchemlyb/parsing/parquet.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+
+from . import _init_attrs
+
+
+@_init_attrs
+def extract_u_nk(path, T):
+    r"""Return reduced potentials `u_nk` from a pandas parquet file.
+
+    The parquet file should be serialised from the dataframe output
+    from any parser with command
+    (`u_nk_df.to_parquet(path=path, index=True)`).
```

Comment:
Need double back-ticks for proper code markup.
```suggestion
    (``u_nk_df.to_parquet(path=path, index=True)``).
```